src/test/scala/io/confluent/examples/streams/ProbabilisticCountingScalaIntegrationTest.scala

```diff
@@ -94,29 +94,19 @@ class ProbabilisticCountingScalaIntegrationTest extends AssertionsForJUnit {
       p
     }
 
-    val builder: StreamsBuilder = new StreamsBuilder()
+    val builder = new StreamsBuilder
     val cmsStoreName = "cms-store"
     builder.addStateStore(createCMSStoreBuilder(cmsStoreName))
 
     // Read the input from Kafka.
-    val textLines: KStream[Array[Byte], String] = builder.stream(inputTopic)
-
-    // Scala-Java interoperability: to convert `scala.collection.Iterable` to  `java.util.Iterable`
-    // in `flatMapValues()` below.
-    import collection.JavaConverters.asJavaIterableConverter
+    val textLines: KStream[Array[Byte], String] = builder.stream[Array[Byte], String](inputTopic)
```

Comment:
Do we need to specify the type twice?

Reply:
The left-hand side is not needed.  But it's often educational when *reading* the source code that you can quickly see the result type of an operation without having to consult your IDE (or API docs).

Reply:
But yes, we could also just write:
```scala
val textLines = builder.stream[Array[Byte], String](inputTopic)
```